jwst/regtest/test_NIRcam_level2a.py

```diff
@@ -0,0 +1,66 @@
+""" Test for the detector1 pipeline using NIRCam data. This should take an uncal
+    file and generate the stage 1 FITS files (rate, ratints, and trapsfilled)."""
+import os
+from glob import glob
+import pytest
+
+from astropy.io.fits.diff import FITSDiff
+from jwst.pipeline import Detector1Pipeline
+
+@pytest.mark.bigdata
+def test_NIRcam_stage1_rate(rtdata, fitsdiff_default_kwargs, _jail):
```

Comment:
This test is redundant; this case is also covered by test_NIRCam_stage1_completion. The intent of this test in the template was to demonstrate running just a single check.  Everyone has been doing this.